src/engine/GameObjects/MapNode.cxx

```diff
@@ -63,11 +63,12 @@ void MapNode::setTileID(const std::string &tileID, const Point &origCornerPoint)
       //TODO: we need to modify neighbors TileTypes to Shore.
       // no break on purpose.
     case Layer::ROAD:
```

Comment:
Previous version of this had roads demolish building layer so I just want to make sure, is this something you intentionally took out here so that roads no longer replace buildings on a node?

Reply:
This was because the Flora tiles were on the BUILDINGS layer before. I moved them to Flora (which needs to be demolished upon placing) 
Roads should not demolish buildings.